package.json

```diff
@@ -378,6 +381,24 @@
                 "description": "Create a new Draft project",
                 "category": "Draft"
             },
+            {
+                "command": "extension.draftCreateDebug",
+                "title": "Create (debug pack)",
+                "description": "Create a new Draft project using the Node Debug pack",
+                "category": "Draft"
+            },
+            {
+                "command": "extension.draftUpDebug",
+                "title": "up --debug",
```

Comment:
`Up (Debug)` perhaps?  We don't usually use the CLI command quite so literally. *grin*